install_config/router/default_haproxy_router.adoc

```diff
@@ -952,6 +952,59 @@ level deep. So in this example, the certificate would be valid for
 _a.cloudapps.example.com_ but not for _a.b.cloudapps.example.com_.
 ====
 
+To manually redeploy the router certs:
+
+. Check to see if a secret containing the router default certs is already added to the router. 
++
+----
+# oc volumes dc/router 
+
+deploymentconfigs/router
+  secret/router-certs as server-certificate
+    mounted at /etc/pki/tls/private
+----
++
+If this is already configured skip to step 3 and just overwrite the secret. 
+
+. Make sure that you have a default cert directory set for the following variable DEFAULT_CERTIFICATE_DIR 
++
+----
+# oc env dc/router --list
+
+DEFAULT_CERTIFICATE_DIR=/etc/pki/tls/private
+----
++
+If not set set the variable to the following. 
++
+----
+oc env dc/router DEFAULT_CERTIFICATE_DIR=/etc/pki/tls/private
+----
+
+. Overwrite or create a router certificate secret. 
++
+----
+$ cat custom-router.crt custom-ca.crt > custom-router.pem 
+----
++
+To overwrite the secret, run the following command:
++
+---- 
+# oc secrets new router-certs tls.crt=custom-router.pem custom-tls.key=router.key -o json --type='kubernetes.io/tls' --confirm | oc replace -f -
```

Comment:
`tls.cert=custom-router.crt` instead of `tls.crt=custom-router.pem`
`tls.key` instead of `custom-tls.key`